blog/2021-11-04-introducing-pwa.md

```diff
@@ -0,0 +1,168 @@
+---
+slug: introducing-pwa
+title: Introducing PWA in the App Platform
+author: Kai Vandivier
+author_url: https://github.com/KaiVandivier
+author_image_url: https://avatars.githubusercontent.com/u/49666798?v=4
+tags: [app platform, pwa, announcement]
+---
+
+The DHIS2 App Platform now supports PWA capabilities in apps made with the platform! The Dashboards App will be the first core app to take advantage of these features to enable offline capability, and it will be used as an example in this article to describe the details of these features.
```

Comment:
I noticed that "Dashboards App" and "Dashboard App" (L108) are both used in this article, and "Dashboard" (singular) is preferred:

- https://github.com/dhis2/dashboard-app
- https://github.com/dhis2/dashboard-app/blob/master/package.json#L2
- https://github.com/dhis2/dashboard-app/blob/master/package.json#L4
- https://github.com/dhis2/dashboard-app/blob/master/d2.config.js#L4

Historically we did have a "Dashboards" app, but that was only because the name "Dashboard" was already in use. This was solved a while ago and now we only have the "Dashboard" app.

Reply:
Gotcha, thanks!